src/args.rs

```diff
@@ -0,0 +1,47 @@
+#[derive(Debug, PartialEq, Eq)]
+pub struct Args {
+    pub command: String
+}
+
+impl Args {
+    pub fn parse(raw_args: Vec<String>) -> Result<Args, String> {
+        match raw_args.len() {
+            0 => Err(String::from("Please pass remote command.")), // TODO more user friendly message, for now it's consistent with Bash version.
```

Comment:
FWIW, `String::from(x)` is equivalent to `x.into()`.

Reply:
I'm not aware of any convention here. `From` and `Into` are duals to each other in some sense.
You are free to use either `String::from(x)`, `x.to_string()`, `x.to_owned()` and `x.into()` and they will do the same thing, but I have an impression that `x.into()` is more common.

Reply:
Damn Rust, String is most confusing type so far, so many options jeez (and I haven't really used `Cow` yet btw)

Let's stick to `from()` for now since docs use it? I'd probably get a sense of convention as I read/write more code, googling didn't help much, I might also ask this later on /r/rust sub to get a sense of what community thinks about it :)